source/brailleDisplayDrivers/noBraille.py

```diff
@@ -16,3 +16,9 @@ class BrailleDisplayDriver(braille.BrailleDisplayDriver):
 	@classmethod
 	def check(cls):
 		return True
+
+	def _get_numCells(self):

```

Comment:
Interesting approach. I think I'd rather see setDisplayByName update the braille viewers state, including the size of the window. I don't think the current approach solves everything @JulienCochuyt came up with.